community/cpio.hpkg

```diff
@@ -0,0 +1,31 @@
+(use ../prelude)
+(import ../core)
+
+(defsrc cpio-src
+  :url
+  "https://ftp.gnu.org/gnu/cpio/cpio-2.13.tar.gz"
+  :hash
+  "sha256:e87470d9c984317f658567c03bfefb6b0c829ff17dbf6b0de48d71a4c8f3db88")
+
+(defpkg cpio
+  :builder
+  (fn []
+    (os/setenv "PATH"
+               (join-pkg-paths ":" "/bin"
+                               [core/awk
+                                core/coreutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/sed]))
+    (os/setenv "CFLAGS" *default-cflags*)
+    (os/setenv "LDFLAGS" *default-ldflags*)
+    #
```

Comment:
Are these empty comments intentional?

Reply:
Yes, I find they improve readability and I don't like whitespace.

Reply:
Hmm not sure how I feel about that. How badly does white space annoy you? :P

Reply:
Ha ha ha.

So when you saw the blank comment, I imagine you wondered what it was doing there.

I have that type of reaction to whitespace in a function or macro body in lisps.  I wonder why there is space.

If it is a problem to have a blank comment, I can insert text afterwards.